spec/spec_helper_acceptance.rb

```diff
@@ -1,11 +1,6 @@
+# This file is completely managed via modulesync
```

Comment:
I'd keep it as it was. The default is really only 2 lines so IMHO there's no benefit in syncing it. Much cleaner to just do the node setup once than to do it in every single test.